### DISCUSSION TITLE
k8s cloud native storage longhorn에 대해 알아보자

### DISCUSSION THREAD

**Discussion by gusah009** [BDD-CLUB/challenge-every-week-posting#365] [1. 개발 관련 글 모음]
그만 알아보자.

부서에서 관련 일을 하면서 longhorn에 대해 써 볼 일이 생겨서 공부겸 정리해봤습니다!
비글챌 취지 좋네요... BDD에 오래 있었는데 왜 이제 알았나 싶네요 허허

> README에 제 이름도 넣어주세요 ㅠㅠ 조쟁이도...
>
> <img width="880" alt="image" src="https://github.com/BDD-CLUB/challenge-every-week-posting/assets/26597702/a6501486-682e-4c8a-847d-72cbd2162462">

https://gusah009.github.io/etc/2024-03-08-longhorn/

**Comment by jojaeng2** [comment]
<img width="1512" alt="image" src="https://github.com/BDD-CLUB/challenge-every-week-posting/assets/76645095/d1b66e46-b391-4b0e-b574-52c54fb864a8">
선생님, 블로그가 깨지는것 같습니다

**Reply by gusah009** [reply]
놀랍게도 정상입니다. 스크롤 왔다 갔다 하면서 시간 지나면 정상으로 돌아와요 ㅋㅋㅋㅋㅋㅋ
